autospec/git.py

```diff
@@ -95,6 +96,23 @@ def commit_to_git(path):
     call("git rm insecure_build", check=False, stderr=subprocess.DEVNULL, cwd=path)
     call("git rm conservative_flags", check=False, stderr=subprocess.DEVNULL, cwd=path)
 
+    # add a gitignore
+    ignorelist = [
+        ".*~",
+        "*~",
+        "*.swp",
+        ".repo-index",
+        "*.log*",
+        "*.tar*",
+        "*.tgz",
+        "*.zip",
+        "commitmsg",
+        "results/",
+        ""
+    ]
+    write_out(os.path.join(path, '.gitignore'), '\n'.join(ignorelist))
```

Comment:
Write it every time, even if it already exists?

Reply:
Yes. My intention was for autospec to own the file, but I can avoid overwriting the file if you think it should be user-controlled after it is first written.

Reply:
No I don't think it should be user controlled. We'll end up with different files checked in depending on who runs autospec. This is fine.